packages/js/product-editor/src/components/block-editor/block-editor.tsx

```diff
@@ -95,13 +100,16 @@ export function BlockEditor( {
 
 	useLayoutEffect( () => {
 		setupEditor( product, {}, _settings?.template );
```

Comment:
This line is breaking the editor cause the product does not contains a prop `content.raw` so we need to skip this or implement something similar.

Reply:
Good catch!

Reply:
Thanks @mdperez86, this seemed to only be a problem for new products as it checked for the `auto-draft` status.
Although the way we were using it, we are better off using `setupEditorState` which `setupEditor` uses internally.

This should be fixed now as part of: https://github.com/woocommerce/woocommerce/pull/37206/commits/abb50e839ea898682b7ff153f4f1e36d8577d49f